charts/renku/requirements.yaml

```diff
@@ -18,7 +18,7 @@ dependencies:
 - name: renku-graph
   alias: graph
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: "0.29.3"
```

Comment:
currently, version `0.30+` is not compatible with the UI. Should we wait for that to be addressed before bumping the version on graph?

Reply:
this version was already bumped by renku-graph bot before the release. cc-ing @jachro

Reply:
it's on `dev` but maybe we can skip pinning the version here, otherwise someone pulling the master branch and trying to update his environment may have problems -- even if it's not a huge incompatibility

Reply:
yes, I also think the versions of components that have dependencies should be updated in a synchronous way.
I suppose @jachro had a discussion about this with @rokroskar ?